spring-integration-rsocket/src/main/java/org/springframework/integration/rsocket/outbound/RSocketOutboundGateway.java

```diff
@@ -72,33 +72,33 @@
  */
 public class RSocketOutboundGateway extends AbstractReplyProducingMessageHandler {
 
+	@SuppressWarnings("NullAway.Init")
```

Comment:
I don't believe in this.
We do always set this from ctor.
What do I miss, please?